sdk/core/azure-core/src/main/java/com/azure/core/models/ResponseError.java

```diff
@@ -35,8 +40,8 @@ public final class ResponseError {
      * @param message the error message of this error.
      */
     @JsonCreator
-    public ResponseError(@JsonProperty(value = "code", required = true)String code,
-                         @JsonProperty(value = "message", required = true)String message) {
+    public ResponseError(@JsonProperty(value = "code", required = true) String code,
```

Comment:
We don't want `required = true` set for any response models. See https://github.com/Azure/autorest.java/issues/2169

Reply:
I'll update that